dir.props

```diff
@@ -452,6 +453,12 @@
         <NuGetTargetMoniker>.NETCoreApp,Version=v1.0</NuGetTargetMoniker>
       </PropertyGroup>
     </When>
+    <When Condition="'$(TargetGroup)'=='netcoreapp1.1'">
```

Comment:
I did this in dev/api branch already here: https://github.com/dotnet/corefx/blob/dev/api/dir.props#L524